### DISCUSSION TITLE
[Bug]: mjpeg API 404: Frigate 0.15 dev

### DISCUSSION THREAD

**Discussion by jasonmadigan** [blakeblackshear/frigate#14131] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Have been trying the 0.15 builds for a while now. Today I updated to the latest build, and am seeing 404s from requests to the old MJPEG camera APIs. Maybe they have moved.

e.g:  http://frigate:5001/api/somecamera?h=480&motion=1&bbox=1&timestamp=1

{"detail":"Not Found"}

### Steps to reproduce

1.
2.
3.
...

### Version

0.15.0-dev

### Frigate config file

```yaml
n/a
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant Frigate log output

```shell
N/a
```

### Relevant go2rtc log output

```shell
N/a
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

n/a

### Screenshots of the Frigate UI's System metrics pages

N/a

**Comment by hawkeye217** [comment]
Major changes were made to the API for 0.15, we'll take a look at this one. Thanks.

**Comment by NickM-27** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/14138

**Reply by jasonmadigan** [reply]
awesome, thanks!
